manim/mobject/mobject.py

```diff
@@ -2600,25 +2643,31 @@ def init_size(num, alignments, sizes):
             buff_x = buff_y = buff
 
         # Initialize alignments correctly
-        def init_alignments(alignments, num, mapping, name, dir_):
+        def init_alignments(
+            alignments: str | None,
+            num: int,
+            mapping: dict[str, Point3D],
```

Comment:
The values of this dictionary refer to alignment directions, so they're more properly represented by vectors.

As this is an internal function, I suggest renaming this parameter to a more descriptive name as well:
```suggestion
            char_to_direction: dict[str, Vector3D],
```